html/modules/custom/reliefweb_reporting/src/Commands/ReliefWebReportingCommands.php

```diff
@@ -760,6 +772,44 @@ public function exportReportData(
       fclose($file);
     }
 
+    if ($upload) {
+      $client = new Google\Client();
+      if (getenv('GOOGLE_APPLICATION_CREDENTIALS')) {
+        $client->useApplicationDefaultCredentials();
+      } else {
+        $this->logger->error('Error: No credentials defined in the GOOGLE_APPLICATION_CREDENTIALS environment.');
+        return FALSE;
+      }
+
+      $client->setApplicationName("Reliefweb Reports Data Uploader");
```

Comment:
Maybe we should use the Drupal http UA string instead?

Reply:
Doesn't really matter